src/Tasks/Microsoft.CSharp.CurrentVersion.targets

```diff
@@ -234,92 +234,80 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
        <!-- Condition is to filter out the _CoreCompileResourceInputs so that it doesn't pass in culture resources to the compiler -->
        <!-- Don't run analyzers for Csc task on XamlPrecompile pass, we only want to run them on core compile. -->
-    <Csc Condition="'%(_CoreCompileResourceInputs.WithCulture)' != 'true'"
-         AdditionalLibPaths="$(AdditionalLibPaths)"
-         AddModules="@(AddModules)"
-         AdditionalFiles="@(AdditionalFiles)"
-         AllowUnsafeBlocks="$(AllowUnsafeBlocks)"
-         AnalyzerConfigFiles="@(EditorConfigFiles)"
-         Analyzers="@(Analyzer)"
-         ApplicationConfiguration="$(AppConfigForCompiler)"
-         BaseAddress="$(BaseAddress)"
-         CheckForOverflowUnderflow="$(CheckForOverflowUnderflow)"
-         ChecksumAlgorithm="$(ChecksumAlgorithm)"
-         CodeAnalysisRuleSet="$(ResolvedCodeAnalysisRuleSet)"
-         CodePage="$(CodePage)"
-         DebugType="$(DebugType)"
-         DefineConstants="$(DefineConstants)"
-         DelaySign="$(DelaySign)"
-         DisabledWarnings="$(NoWarn)"
-         DisableSdkPath="$(DisableSdkPath)"
-         DocumentationFile="@(DocFileItem)"
-         EmbedAllSources="$(EmbedAllSources)"
-         EmbeddedFiles="@(EmbeddedFiles)"
-         EmitDebugInformation="$(DebugSymbols)"
-         EnvironmentVariables="$(CscEnvironment)"
-         ErrorEndLocation="$(ErrorEndLocation)"
-         ErrorLog="$(ErrorLog)"
-         ErrorReport="$(ErrorReport)"
-         Features="$(Features)"
-         InterceptorsPreviewNamespaces="$(InterceptorsPreviewNamespaces)"
-         FileAlignment="$(FileAlignment)"
-         GeneratedFilesOutputPath="$(CompilerGeneratedFilesOutputPath)"
```

Comment:
I assume this isn't relevant to the precompilation step but would like confirmation.

Reply:
Unfortunately everything but `FileAlignment` is necessary. The rest all impact compilation success / failure.